superset-frontend/src/views/components/MenuRight.tsx

```diff
@@ -439,4 +438,38 @@ const RightMenu = ({
   );
 };
 
-export default RightMenu;
+const RightMenuWithQueryWrapper: React.FC<RightMenuProps> = props => {
+  const [, setQuery] = useQueryParams({
+    databaseAdded: BooleanParam,
+  });
+
+  return <RightMenu setQuery={setQuery} {...props} />;
+};
+
+class RightMenuErrorWrapper extends React.PureComponent<RightMenuProps> {
```

Comment:
Can we add some comments here? Others may be curious as to why an additional error wrapper is needed.